core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneOptimized.java

```diff
@@ -125,6 +130,26 @@ private DaoGeneOptimized () {
         }
     }
 
+    private void clearCache()
```

Comment:
This will also make it possible to update your set of genes via an import script (like `ImportGeneData.java` - although this class needs to be improved) and just call recache instead of having to restart the webserver.